index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+     <title>Capstone project</title>
+  </head>
+  <body>
+    <header class="head">
+      <div class="homepage">
+        <div class="nav-list-one">
+          <ul class="list">
+            <li>
+              <a class="header-icon" href="#"
+                ><i class="fa-brands fa-facebook-f"></i
+              ></a>
+            </li>
+            <li>
+              <a class="header-icon" href="#"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li>English</li>
+            <li>My page</li>
+            <li>Log out</li>
+          </ul>
+        </div>
+<div class="all">
+          <nav class="nav-bar">
+            <a class="nav-img" href="./index.html"><img id="nav-img" src="./images/RO.jpeg" alt=""></a>
```

Comment:
- [ ] Here also, remember to include a descriptive meaning to the image using the `alt`. i made a detailed review of the summary, please check it out 👍